src/app/project-admin/component/project-admin.component.html

```diff
@@ -0,0 +1,187 @@
+                                    <th scope="col"
+                                        class="px-3 py-2 text-center text-xs font-medium uppercase tracking-wide text-gray-500">
+                                    </th>
+                                </tr>
+                            </thead>
+                            <tbody class="divide-y divide-gray-200">
+
+                                <tr *ngFor="let tokenContainer of personalAccessTokens; let j = index"
+                                    [ngClass]="j%2!=0 ? 'bg-gray-50':'bg-white'">
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        {{tokenContainer.name}}
+                                    </td>
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        {{tokenContainer.scope}}
+                                    </td>
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        {{tokenContainer.createdBy}}
+                                    </td>
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        {{tokenContainer.createdAt}}
+                                    </td>
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        {{tokenContainer.expiresAt}}
+                                    </td>
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        {{tokenContainer.lastUsed}}
+                                    </td>
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        <label [for]="'my-modal-'+tokenContainer.id"
+                                            class="modal-button btn btn-sm btn-ghost px-0">
+
+                                            <svg xmlns="http://www.w3.org/2000/svg"
+                                                class="inline-block w-6 h-6 stroke-current text-red-700" fill="none"
+                                                viewBox="0 0 24 24" stroke="currentColor">
+                                                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                                                    d="M19 7l-.867 12.142A2 2 0 0116.138 21H7.862a2 2 0 01-1.995-1.858L5 7m5 4v6m4-6v6m1-10V4a1 1 0 00-1-1h-4a1 1 0 00-1 1v3M4 7h16" />
+                                            </svg>
+                                        </label>
+                                        <input type="checkbox" [id]="'my-modal-'+tokenContainer.id"
```

Comment:
we dont use the modal in this form anymore (lina is working on changing all occurrences in a different branch)

we now use kern-modal and only one per logical bound (so not one for every table entry only one in general -> i can show you how or change it if you prefer)